### DISCUSSION TITLE
how to convert image to mesh?

### DISCUSSION THREAD

**Discussion by wild-fruits** [bevyengine/bevy#13634] [Q&A]
how to convert image to mesh?

**Comment by Fidius-jko** [comment]
We can't conver image to mesh. Maybe you want to show sprite? Use standart sprite for this or check source code of sprite. That is example for sprite: https://bevyengine.org/examples/2D%20Rendering/sprite/.
In 3d https://bevyengine.org/examples/3D%20Rendering/texture/

**Reply by wild-fruits** [reply]
Thanks for your guidance sir
